test/filters/filter.spec.js

```diff
@@ -207,4 +207,51 @@ describe('Directive:  pfFilter', function () {
 
     expect(element.find('.pf-table-view-selected-label').text()).toContain('0 of 10 selected');
   });
+
+  it('should show results inline when specified', function() {
```

Comment:
To be truly well rounded, each of these three tests need a negative test.  Ex:  "should not show results inline when not specified"  ```expect(element.find('.filter-pf.inline-filter-pf').length).toBe(0);```

Reply:
Added negative tests

Reply:
You did?  I don't see them?